app/controllers/archive_faqs_controller.rb

```diff
@@ -18,7 +18,20 @@ def index
 
   # GET /archive_faqs/1
   def show
-    @archive_faq = ArchiveFaq.find_by_slug(params[:id])
+    @questions = []
+    if params[:language_id] == "en"
+      @archive_faq = ArchiveFaq.find_by_slug(params[:id])
```

Comment:
This line gets called for both conditions, better to move it above the "if/else"